tests/main.cpp

```diff
@@ -707,3 +707,19 @@ TEST_CASE("account9") {
   CHECK_EQ(leverage.to_double(), 3.919442838288937);
   CHECK_FALSE(mangoAccount.isLiquidatable(mangoGroup, mangoCache));
 }
+
+TEST_CASE("getVersion") {
+  const solana::Keypair keyPair = solana::Keypair::fromFile(KEY_PAIR_FILE);
+  const auto connection = solana::rpc::Connection(solana::DEVNET);
+  // get Version
+  const auto version = connection.getVersion();
+  CHECK_GT(version.solana_core.length(), 0);
+}
+
+TEST_CASE("getFirstAvailableBlock") {
+  const solana::Keypair keyPair = solana::Keypair::fromFile(KEY_PAIR_FILE);
+  const auto connection = solana::rpc::Connection(solana::DEVNET);
+  // get Version
+  const auto firstAvailableBlock = connection.getFirstAvailableBlock();
```

Comment:
Can also add a test like
```
const auto slot = connection.getSlot(); // get finalized slot.
const auto firstAvailableBlock = connection.getFirstAvailableBlock();
CHECK_GT(firstAvailableBlock, slot);
sleep(2); // sleep for 2 seconds
const auto availableBlock = connection.getFirstAvailableBlock();
CHECK_GT(availableBlock, firstAvailableBlock);
```

Reply:
Yes this does not really work because getAvailableBlock is the block in the ledger. I guess you can keep it like it is.

Reply:
Ok